app/_src/gateway-operator/get-started/konnect/create-route.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Create a Route
+content-type: tutorial
+book: kgo-konnect-get-started
+chapter: 3
+---
+
+{:.note}
+> The following examples require you to have `jq` installed on your machine. You can use the [Konnect UI](https://cloud.konghq.com) to configure services and routes if you cannot install `jq`.
+
+Your `DataPlane` should now be running, but there are no routing rules to tell {{ site.base_gateway }} how to proxy traffic. In this tutorial we will use the Konnect API to configure and test routing.
```

Comment:
```suggestion
After you deploy the`DataPlane` node it is  connecetd to the your cluster plane, but there are no routing rules  for {{ site.base_gateway }} to proxy traffic. You can use Konnect API to configure and test routing.
```

Reply:
Neither option feels right here. How about:

```suggestion
Once your `DataPlane` is deployed it will receive configuration from Konnect, but there are no routing rules available for {{ site.base_gateway }} to proxy traffic. You can use Konnect API to configure and test routing.
```

Reply:
> Neither option feels right here. How about:

How about:

After your 'DataPlane`is deployed it receives configuration from Konnect, but there are no routing rules available for {{ site.base_gateway }} to proxy traffic. You can use Konnect API to configure and test the routing.